assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Pagination.js

```diff
@@ -60,19 +62,47 @@ const Pagination = ( { tab } ) => {
 		return 0;
 	} );
 
+	const trackPage = useCallback(
+		( direction ) => {
+			const eventMap = {
+				forward: {
+					'new-ideas': 'new_page_advance',
+					'save-ideas': 'saved_page_advance',
+					'draft-ideas': 'draft_page_advance',
+				},
+				back: {
+					'new-ideas': 'new_page_return',
+					'save-ideas': 'saved_page_return',
+					'draft-ideas': 'draft_page_return',
+				},
+			};
+			const event = eventMap[ direction ][ tab ];
+
+			trackEvent( IDEA_HUB_GA_CATEGORY_WIDGET, event, page );
+		},
+		[ tab, page ]
+	);
+
 	const { setValue } = useDispatch( CORE_UI );
 
-	const handlePrev = useCallback( () => {
-		if ( page > 1 ) {
-			setValue( uniqueKey, page - 1 );
-		}
-	}, [ page, setValue, uniqueKey ] );
+	const handlePrev = useCallback(
+		( shouldTrackEvent = true ) => {
+			if ( page > 1 ) {
+				setValue( uniqueKey, page - 1 );
+			}
+			if ( shouldTrackEvent ) {
+				trackPage( 'back' );
+			}
```

Comment:
We should call the `trackPage` function only if the current page allows the user to go back. In other words, if the current page is greater than `1`.

Reply:
Great catch - this should be in the `if` block above.